packages/experiments/src/components/Microfeedback/Microfeedback.view.tsx

```diff
@@ -0,0 +1,178 @@
+import { IMicrofeedbackState } from './Microfeedback.state';
+
+import { initializeIcons } from '@uifabric/icons';
+
+const theme = getTheme();
+
+const microfeedbackStyles: IStackStyles = {
+  root: [
+    {
+      margin: 8,
+      float: 'right'
+    }
+  ]
+};
+
+const microfeedbackItemStyles: IButtonStyles = {
+  root: [
+    theme.fonts.small,
+    {
+      width: '100%',
+      color: theme.palette.neutralPrimary,
+      fontWeight: FontWeights.light,
+      fontSize: 12
+    }
+  ]
+};
+
+const styles = mergeStyleSets({
+  title: [
+    theme.fonts.medium,
+    {
+      margin: 8,
+      color: theme.palette.neutralPrimary,
+      fontWeight: FontWeights.semibold
+    }
+  ],
+  body: [
+    theme.fonts.small,
+    {
+      margin: 6,
+      color: theme.palette.neutralPrimary,
+      fontWeight: FontWeights.regular
+    }
+  ]
+});
+
+class MicrofeedbackViewComponent extends React.Component<IMicrofeedbackProps, IMicrofeedbackState> {
+  // ref's will be linked to each of the icons for callout placement
+  private dislikeRef = React.createRef<HTMLDivElement>();
+  private likeRef = React.createRef<HTMLDivElement>();
+
+  constructor(props: IMicrofeedbackProps) {
+    super(props);
+
+    initializeIcons();
+
+    this.state = {
+      // initial state of icons is neutral and followup is not visible
+      vote: 0,
+      isFollowupVisible: false
+    };
+  }
+
+  public render() {
+    const likeIcon = this.state.vote > 0 ? 'LikeSolid' : 'Like';
+    const dislikeIcon = this.state.vote < 0 ? 'DislikeSolid' : 'Dislike';
+    const hideThumbsDownCallout = this.state.vote !== -1 || !this.state.isFollowupVisible;
+    const hideThumbsUpCallout = this.state.vote !== 1 || !this.state.isFollowupVisible;
+    const onCalloutDismiss = this._onCalloutDismiss.bind(this);
+
+    return (
+      <div>
+        <Stack horizontal styles={microfeedbackStyles}>
+          <div ref={this.likeRef}>
+            <IconButton menuIconProps={{ iconName: likeIcon }} title={this.props.thumbsUpTitle} onClick={this._vote.bind(this, 1)} />
+          </div>
+          <div ref={this.dislikeRef}>
+            <IconButton menuIconProps={{ iconName: dislikeIcon }} title={this.props.thumbsDownTitle} onClick={this._vote.bind(this, -1)} />
+          </div>
+        </Stack>
+        {this.props.ThumbsUpQuestion ? (
+          <Callout
+            hidden={hideThumbsUpCallout}
+            role="alertdialog"
+            gapSpace={0}
+            target={this.likeRef.current}
+            setInitialFocus={true}
+            onDismiss={onCalloutDismiss}
+          >
+            <div>
+              <FocusZone direction={FocusZoneDirection.vertical}>
+                <p className={styles.title}>{this.props.ThumbsUpQuestion.question}</p>
+                <List items={this.props.ThumbsUpQuestion.options} onRenderCell={this._onRenderCalloutItem} />
+              </FocusZone>
+            </div>
+          </Callout>
+        ) : (
+          <div />
```

Comment:
Can you just return `null` here? It's a valid render output when you don't want to render anything.

Reply:
Fixed